### PR TITLE
add kannada language support

### PR DESCRIPTION
**Pull request by akki2825**
### Changes proposed in this pull request:

Adds Kannada language support.

### Status

- [X] READY
- [ ] HOLD
- [ ] WIP (Work-In-Progress)

### CODE REVIEW

**Comment by akki2825** [comment]
@erozqba Tests are failing for `test_cli.py`

**Comment by akki2825** [comment]
@erozqba tests are failing for `test_cli.py`. could you guide me here?

**Comment by kkonieczny** [comment]
@akki2825 can you rebase your branch?

**Comment by akki2825** [comment]
@kkonieczny thanks for the reply. I did update, its passing on py36 env but failing on others. Not sure what I'm missing missing.

File: num2words/__init__.py [hunk 1]
```diff
@@ -101,8 +102,6 @@ def num2words(number, ordinal=False, lang='en', to='cardinal', **kwargs):
     if lang not in CONVERTER_CLASSES:
         raise NotImplementedError()
     converter = CONVERTER_CLASSES[lang]
-    if isinstance(number, str):
-        number = converter.str_to_number(number)
```
Comment (lepisma) [inline num2words/__init__.py]: Hey @akki2825 you are missing this section. With this conditional, the tests pass.

Comment (akki2825) [reply num2words/__init__.py]: Thank you @lepisma

**Comment by sarahberanek** [comment]
I am very much interested in this. How is the status of this pull request?

**Comment by lepisma** [comment]
@sarahberanek I don't know kannada but I believe there were/are certain inaccuracies in few numbers or number ranges. We use a fork (not PRed here) which _probably_ has those fixed. @akki2825 can you check [this fork's](https://github.com/Vernacular-ai/num2words/tree/indic-languages) code and say something about the changes?

**Comment by sarahberanek** [comment]
@lepisma ah that is interesting! Thank you for the info. Any way that we can fix the bugs here and merge a correct version into the official master?

**Comment by akki2825** [comment]
@lepisma thanks for the info. I have missed a condition in number ranges. @sarahberanek, I'll make the necessary changes and get back by tomorrow.

**Comment by sarahberanek** [comment]
Super, thanks!

**Comment by erozqba** [comment]
@akki2825 Thanks a lot for this contribution! I have tried to resolve a conflict on your PR, but my changes were not correct. Please check Travis and Coveralls responses. Could you also add more tests? You could check on Travis the missed lines.

I will be doing a new release this weekend, it will be great to include this on the release.

**Comment by erozqba** [comment]
@akki2825 I will do a release tomorrow, do you think you will have time to finish this?

**Comment by erozqba** [comment]
@akki2825 thanks a lot! We are almost there just missing a couple of tests to cover these lines https://coveralls.io/builds/23334032/source?filename=num2words%2Flang_KN.py 
I can't merge if Coveralls is failing, :pray:

**Comment by akki2825** [comment]
Hey, I was a bit occupied. Apologies for the delay. @erozqba check now.

**Comment by erozqba** [comment]
@akki2825 I forgot to mention before, could you update the README.md to add this new language as part of your PR?

**Comment by akki2825** [comment]
@erozqba done.

**Comment by sarahberanek** [comment]
> @akki2825 I will do a release tomorrow, do you think you will have time to finish this before the release so we can make @sarahberanek happy?

Haha, yes thanks @akki2825 and @erozqba! I am happy now :smile:

**Pull request merged**